src/EditorFeatures/Core/Implementation/IntelliSense/Completion/UseAsyncCompletionOptionDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.using System;
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion
+{
+    [Export(typeof(EditorOptionDefinition))]
```

Comment:
I'm not sure if it's appropriate to export the option definition twice. 
I'm exporting the definition in my PR so that I can write the value.
However, to read it, we identify it by string `UseAsyncCompletion` without the need for the definition.